### DISCUSSION TITLE
Report with Restricted Data Token

### DISCUSSION THREAD

**Discussion by valle-gp** [abuzuhri/Amazon-SP-API-CSharp#334] [General]
Hi,

I have a nedd to download a report that requires the Restricted Data Token (RDT) (see https://github.com/amzn/selling-partner-api-docs/issues/563)
The  Amazon-SP-API-CSharp support the RDT but how the request should be formulated? 
There is an axample?

**Comment by valle-gp** [comment]
> Hi,
> 
> I have a nedd to download a report that requires the Restricted Data Token (RDT) (see [amzn/selling-partner-api-docs#563](https://github.com/amzn/selling-partner-api-docs/issues/563)) The Amazon-SP-API-CSharp support the RDT but how the request should be formulated? There is an axample?

Sorry, I try to create the SC_VAT_TAX_REPORT thet si only for European Maerketplace.
TRhnaks

**Reply by abuzuhri** [reply]
WE HAVE SAMPLE CODE FOR RDT
